.azure-pipelines/ci-build.yml

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - master
+pr: none
```

Comment:
Are we only triggering the build when merging and not on PR to the `vnext` branch?

Reply:
Since I didn't see any previews, I figured you only want to release when you merge to master. You tell me which branches should trigger and represent the released NuGet.